aries-site/src/layouts/navigation/Search.js

```diff
@@ -27,8 +27,6 @@ export const Search = ({ focused, setFocused }) => {
   const size = useContext(ResponsiveContext);
   const [value, setValue] = useState('');
   const [suggestions, setSuggestions] = useState(allSuggestions);
-
-  const boxRef = useRef();
   const inputRef = useRef();
```

Comment:
Since you took out `const boxRef = useRef();` was this still needed ? maybe just change up comment

Reply:
Good catch, I can actually remove a few functions there.